bridges/bridge/bridgeExecutor.go

```diff
@@ -5,9 +5,9 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/ElrondNetwork/elrond-eth-bridge/bridges"
```

Comment:
I would have renamed this to something like:
`bridges/ethElrond/bridgeExecutor.go` as to specify that the bridge is eth <-> elrond. This naming will be extensible in the future when we need to implement a non-eth bridge with elrond